### ISSUE TITLE
return type of createNativeQuery(String, Class)

### ISSUE THREAD

**Issue by lukasj** [jakartaee/persistence#52]
Hi,

I've got a minor suggestion:

    EntityManager.createNativeQuery(String, Class)

could return a TypedQuery as the result class is specified:

    TypedQuery<T> createNativeQuery(java.lang.String sqlString, java.lang.Class<T> resultClass)

-Adrian

**Comment by m-reza-rahman** [comment]
I don't think this is possible without breaking backwards compatibility and this seems very minor. I suggest just closing it to reduce clutter.

Reza Rahman
Jakarta EE Ambassador, Author, Blogger, Speaker

Please note views expressed here are my own as an individual community member and do not reflect the views of my employer.

**Comment by lprimak** [comment]
I support this for being included in 4 as well
